jbrotli/src/main/java/org/meteogroup/jbrotli/BrotliStreamDeCompressor.java

```diff
@@ -46,7 +46,7 @@ public BrotliStreamDeCompressor() throws BrotliException {
    * @param out output byte array
    * @return length of decompressed byte array
    */
-  public final int deCompress(byte[] in, byte[] out) {
+  public final int[] deCompress(byte[] in, byte[] out) {
```

Comment:
Returning a 2-dim-tuple is difficult to understand and requires a lookup in the docs.
I prefer a concrete Java Object ... e.g. returning a similar one to BrotliStreamDeCompressorResult,
just containing 'bytesConsumed' and 'bytesProduced'?